standard/info/wikis/zula/info.lua

```diff
@@ -24,6 +24,45 @@ return {
 				lightMode = 'Zula Global default allmode.png',
 			},
 		},
+		global = {
+			abbreviation = 'Global',
+			name = 'Zula Global',
+			link = 'Zula Global',
+			logo = {
+				darkMode = 'Zula Global default allmode.png',
+				lightMode = 'Zula Global default allmode.png',
+			},
+			defaultTeamLogo = {
+				darkMode = 'Zula Global allmode.png',
+				lightMode = 'Zula Global allmode.png',
+			},
+		},
+		oyun = {
+			abbreviation = 'Oyun',
+			name = 'Zula Oyun',
+			link = 'Zula Oyun',
+			logo = {
+				darkMode = 'Zula Oyun allmode.png',
+				lightMode = 'Zula Oyun allmode.png',
+			},
+			defaultTeamLogo = {
+				darkMode = 'Zula Oyun allmode.png',
+				lightMode = 'Zula Oyun allmode.png',
+			},
+		},
+		europe = {
+			abbreviation = 'Europe',
+			name = 'Zula Europe',
+			link = 'Zula Europe',
+			logo = {
+				darkMode = 'Zula Oyun allmode.png',
```

Comment:
sure that this is the correct file for this game?
it is already used for oyun, now you also use it for europe

Reply:
From what I've been told Europe and Oyun uses the same logo (so two different versions with the same logo)